lua/tests/current_search_paths_spec.lua

```diff
@@ -0,0 +1,39 @@
+local lean = require('lean')
+local helpers = require('tests.helpers')
+local fixtures = require('tests.fixtures')
+
+require('tests.helpers').setup {
+  lsp = { enable = true },
+  lsp3 = { enable = true },
+}
+describe('lean.current_search_paths', function()
+  for kind, path in unpack(fixtures.lean_project.files_it) do
+    it(string.format('returns the paths for %s files', kind), function()
+      vim.api.nvim_command('edit ' .. path)
+      helpers.wait_for_ready_lsp()
+
+      local paths = lean.current_search_paths()
+      -- via its leanpkg.path:
+      assert.has_all(
```

Comment:
We should probably make these tests more specific (i.e. using [exact match with `assert.has_match()`](https://github.com/Julian/lean.nvim/blob/d534ad4bf3e545d5be6e16b7746eff6304421119/lua/tests/helpers.lua#L113)) to catch the bug Gabriel noticed above.

Reply:
It's hard to do that I think without doing something more complex than the actual implementation, given the paths are absolute (intentionally) and the Lean stdlib location will even vary by machine -- do you see an obvious way to do it?

To fix the bug above I need to basically add a second directory (because right now no colons are in the output for the test case) but yeah let me know what you think.

Reply:
No problem, I think what you've just done (checking the suffix using `\n`) is already good enough.